### PR TITLE
Add print_dictionary helper

### PR DESCRIPTION
**Pull request by Hannibal404**

### CODE REVIEW

**Comment by Hannibal404** [comment]
Made the organization membership public

File: drgn_tools/table.py [hunk 1]
```diff
@@ -42,3 +43,37 @@ def print_table(
...
+    out = sys.stdout
+    if outfile and report:
+        out = open(outfile, "a")
+        print("\n", file=out)
+    elif outfile:
+        out = open(outfile, "w")
+
+    for title in dictionary:
+        print(
+            f"{title.ljust(lcol_length)}: {dictionary[title]:<}",
```
Comment (brenns10) [inline drgn_tools/table.py:74]: `{dictionary[title]:<}`
nit: technically, left alignment is the default, so there's no need to specify it

**Review by brenns10 — CHANGES_REQUESTED** [review]
One small nit but otherwise, it looks good to me!

**Review by brenns10 — APPROVED** [review]

**Comment by brenns10** [comment]
Thanks for the quick work unblocking both pull requests!

**Pull request merged**